app.js

```diff
@@ -1,25 +1,32 @@
+//local imports, as requested
+const baseUtil = require('./base');
+const { PlanetWars, MAPS, AIS } = require('./game');
+const { WebsocketView } = require('./socketview');
+
+//require actual packages
 const bodyParser = require('body-parser');
+const _ = require('lodash');
 const express = require('express');
+
+
+//init some stuff
 const app = express();
-const ldKeys = require('lodash/keys');
 app.set('views', __dirname + '/views');
 app.set('view engine', 'nunjs');
 app.use(bodyParser());
-const baseUtil = require('./base');
-
-const {WebsocketView} = require('./socketview');
-
 const server = require('http').Server(app);
 const io = require('socket.io')(server);
-const { PlanetWars, MAPS, AIS } = require('./game');
-
 const games = {};
+//publish static files
```

Comment:
Comments are good but there should 1) be a blank line above them all and 2) be a space after the `//`.